.azuredevops/release.yml

```diff
@@ -113,14 +113,12 @@ extends:
             inputs:
               command: restore
               feedsToUse: select
-              vstsFeed: $(VSTS_FEED_ID)
+              vstsFeed: Office
               includeNuGetOrg: false
               arguments: --runtime ${{ config.runtime }}
           # 1ES PT requires explicit build task for Roslyn analysis. Auto-injected Roslyn task will use build logs from this build.
           - task: DotNetCoreCLI@2
             displayName: Build projects
-            env:

```

Comment:
Since the nuget.config file no longer has this ADO_TOKEN token, we can remove this unnecessary environment variable.